benchmarks/domains/Hardness.py

```diff
@@ -0,0 +1,183 @@
+"""Synthetic function with two continuous and one discrete input."""

```

Comment:
See other PR: Should we already comment here?

Reply:
not ready, deliberately set to draft

Reply:
Ok, then let me resolve here.